openghg/util/_domain.py

```diff
@@ -337,14 +337,22 @@ def align_lat_lon(data: XrDataLikeMatch, domain: str) -> XrDataLikeMatch:
     the 'check_coord_alignment' function.
 
     Args:
-        data: spatial data to be checked. Must have 'lat' and 'lon' dimensions
+        data: spatial data to be checked. Must have 'lat'/'latitude' and 'lon'/'longitude' dimensions
         domain: domain in question. Must be a valid domain in openghg_defs
 
     Returns:
         xarray.DataArray / xarray.Dataset: data with lat, lon ranges aligned to openghg domain.
     """
-
-    data = check_coord_alignment(data=data, domain=domain, coord="lat")
-    data = check_coord_alignment(data=data, domain=domain, coord="lon")
+    dims = list(data.sizes.keys())
```

Comment:
You can probably just use `dims = data.dims`. Right now `data.dims` is a dictionary, but I think it will change to tuple in the future. But e.g. `time in {"time": 1234}` will return true.